### DISCUSSION TITLE
light positioning

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#2123] [Q&A] [answered]
Hey, guys!
Is there a way to change the position of light?
Or to attach it to the camera?

**Comment by elalish** [comment, marked as answer]
All lighting is environmental (IBL). If you want it to be different than the default, simply use your own HDR environment image: https://modelviewer.dev/docs/index.html#entrydocs-lightingandenv-attributes-environmentImage

You can add a directional light by painting in a very bright spot in the HDR.
